packages/components/src/ecosystem_bar/index.module.scss

```diff
@@ -0,0 +1,82 @@
+@import "~nebenan-ui-kit";
+
+// IMPORTANT: global class names dependency
+// 'client/components/layout_header/index.jsx'
+$external-header-wrapper: ".c-layout_header";
+$external-header-panel: ".c-layout_header_panel";
+$external-header-menu: ".c-layout_header-menu-body";
+
+.root {
+  height: 40px;
+  padding: 0 24px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  background: #303030;
```

Comment:
I think color should be a constant. 
We don't have exactly #303030, but maybe we can use $color-gray-00 in here instead?

Reply:
It's a question for Designers, @eliasbraun if you are motivated, ask Leticia, or just keep #303030 (IMHO)

Reply:
Will do!

Reply:
Checked with design, you were right Sergei `$color-gray-00` is the color of choice here

Reply:
Change of plans: since there are considerations to change the navbar color anyway and Daniel worked on that file and chose #303030, we agreed on sticking to that color for now.

Reply:
Maybe add a FIXME or a TODO, when the UI kit get updated to update it to a variable?